interp/src/primitives/combinational.rs

```diff
@@ -226,10 +272,23 @@ comb_primitive!(StdRsh[WIDTH](left: WIDTH, right: WIDTH) -> (out: WIDTH) {
 
 // ===================== Signed Shift Operations ======================
 comb_primitive!(StdSlsh[WIDTH](left: WIDTH, right: WIDTH) -> (out: WIDTH) {
-todo!()
+    let shift_amount = right.as_usize();
```

Comment:
Is there a need to add an assert/panic when the value can't be represented using a usize?

Reply:
The `as_...` methods all have an assert to check validity of the conversion